exporters/jaeger/src/main/java/io/opentelemetry/exporters/jaeger/Adapter.java

```diff
@@ -174,15 +175,19 @@ private Adapter() {}
     switch (value.getType()) {
       case STRING:
         builder.setVStr(value.getStringValue());
+        builder.setVType(Model.ValueType.STRING);
```

Comment:
This was done in a separate PR, can you rebase to remove these changes?

Reply:
yeah I tried to rebase early this morning and flubbed something will fix this now